docs/evm/data-indexers.md

```diff
@@ -10,7 +10,7 @@ When building applications that leverage Flow data, developers have multiple Dat
 
 ## EVM & Cadence
 
-### **SimpleHash**
```

Comment:
These '**' bold weren't affecting these headings and wern't on every heading so removed.